devel/google-glog/Portfile

```diff
@@ -50,3 +50,28 @@ post-destroot {
 }
 
 github.livecheck.regex {([0-9.]+)}
+
+# access to version 0.4.0 until this issue is fixed
+# https://github.com/pytorch/pytorch/issues/58054
+subport ${name}-0.4.0 {
```

Comment:
Should this be named google-glog-0.4 instead? (If a version 0.4.1 were to appear, would we want to update this port to it?)

Reply:
Thanks for the good suggestion. Changed.